pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -54,6 +54,9 @@
   // Pinot controller resource enabled
   PINOT_LEAD_CONTROLLER_RESOURCE_ENABLED("PinotLeadControllerResourceEnabled", true),
 
+  // Number of partitions for which current controller becomes the leader
+  NUMBER_PARTITIONS_CONTROLLER_BECOMES_LEADER("NumberPartitionsControllerBecomesLeader", true),
```

Comment:
Rename to `ControllerLeaderPartitionCount` or `NumControllerLeaderPartitions`

Reply:
Done.